src/main/java/com/ss/ita/google/ui/pages/ExternalPage.java

```diff
@@ -0,0 +1,11 @@
+package com.ss.ita.google.ui.pages;
+
+import org.openqa.selenium.By;
+import static com.ss.ita.google.ui.pages.runnerAndProperties.TestRunner.driver;
+
+public class ExternalPage {
+
+    public boolean isPresentReloadButton() {
+        return driver.findElements(By.xpath("//button[@id='reload-button']")).size() == 1 ? true : false;
```

Comment:
highly inefficient, use direct locator instead of getting the list and checking it's size